controllers/admin/AdminCartsController.php

```diff
@@ -95,15 +95,18 @@ public function __construct()
                 'class' => 'fixed-width-lg',
                 'filter_key' => 'a!date_add'
             ),
-            'id_guest' => array(
+        );
+
+        if (Configuration::get('PS_GUEST_CHECKOUT_ENABLED')) {
+            $this->fields_list['id_guest'] = array(
                 'title' => $this->trans('Online', array(), 'Admin.Global'),
                 'align' => 'text-center',
                 'type' => 'bool',
                 'havingFilter' => true,
                 'class' => 'fixed-width-xs',
-                'icon' => array(0 => 'icon-', 1 => 'icon-user')
```

Comment:
Because, they didn't work : 
![mickael](https://cloud.githubusercontent.com/assets/9074610/18784476/951a8d44-8192-11e6-9d1b-f42756bb162e.png)

And, in this case, **yes or no is better**